parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -81,6 +81,16 @@ pub mod pallet {
 		type MaxAttestationSize: Get<u32>;
 		#[pallet::constant]
 		type MaxValidatorsPerCommittee: Get<u32>;
+		#[pallet::constant]
```

Comment:
I think we should restructure these fork parameters. Otherwise the pallet config will eventually become cluttered with fork parameters.

Could have an enum or a struct like the following (just an idea):
```rust
struct Fork {
  altair: ForkDetails,
  bellatrix: ForkDetails
}

struct ForkDetails { }
```

Reply:
Good point, that would be much cleaner. Done in https://github.com/Snowfork/snowbridge/pull/733/commits/2937528fb2041b9ba365ccba53580051af02ceb2.